python_components/document_inference/document_inference/helpers.py

```diff
@@ -49,11 +49,7 @@ def get_secret(secret_name: str, local_mode: bool) -> str:
 def get_file(url: str, output_path: str) -> str:
     file_name = os.path.basename(url)
     local_path = f"{output_path}/{file_name}"
-    headers = {"User-Agent": "cfa:asap-pdf"}
-    with requests.get(url, headers=headers, stream=True) as response:
-        response.raise_for_status()
-        with open(f"{output_path}/{file_name}", "wb") as file:
-            shutil.copyfileobj(response.raw, file)
+    urllib.request.urlretrieve(url, local_path)
```

Comment:
This change is related to [ASAP-188](https://codeforamerica.atlassian.net/browse/ASAP-188). Can remove this if we want to do more testing.